gobblin-data-management/src/main/java/gobblin/data/management/copy/CopySource.java

```diff
@@ -119,7 +119,8 @@
                 CopyEventSubmitterHelper.DATASET_ROOT_METADATA_NAME, copyableDataset.datasetRoot().toString()));
             workUnit.setProp(SlaEventKeys.DATASET_URN_KEY, copyableDataset.datasetRoot().toString());
             workUnit.setProp(SlaEventKeys.PARTITION_KEY, copyableFile.getFileSet());
-            workUnit.setProp(SlaEventKeys.ORIGIN_TS_IN_MILLI_SECS_KEY, copyableFile.getFileStatus().getModificationTime());
+            workUnit.setProp(SlaEventKeys.ORIGIN_TS_IN_MILLI_SECS_KEY, copyableFile.getOriginTimestamp());
```

Comment:
you don't need to set these properties anymore, the `CopyableFile` is available in the `WorkUnit`, and contains all this information.